source/community/release_notes/sumac/dev_op_release_notes.rst

```diff
@@ -40,7 +57,15 @@ Other Operator Changes
 Deprecations & Removals
 ***********************
 
-
+- In `frontend-app-learner-dashboard <https://github.com/openedx/frontend-app-learner-dashboard>`_ 
+   - support for Optimizely has been removed along with the ProductRecommendations widget.
+   - `DEPR: Optimizely Support <https://github.com/openedx/frontend-app-learner-dashboard/issues/387>`_
+   - Removed the RecommendationsPanel widget
+   - `DEPR: RecommendationsPanel <https://github.com/openedx/frontend-app-learner-dashboard/issues/410>`_
+- In edx-platform:
+   - ``commerce-coordinator`` related code has been removed
+   - `PR: <https://github.com/openedx/edx-platform/pull/35527>`_
+- [UPCOMING] In Teak pre-design-tokens brand packages will no longer be supported. With design tokens, theme authors will instead override core Paragon tokens by defining their own JSON tokens that get deep merged alongside the core Paragon tokens, thus overriding any tokens that were defined by the theme author. See `the associated DEPR ticket for details <https://github.com/openedx/brand-openedx/issues/23>`_.
```

Comment:
We should mention here the fact that the cs_comments_service application is being replaced by forum: 
https://github.com/openedx/cs_comments_service/issues/437
https://github.com/openedx/forum

For users running Tutor, they will automatically switch to the new app. Instructions to migrate data from mongodb to mysql is available in the plugin README https://github.com/overhangio/tutor-forum/ (not yet merged, see [this PR](https://github.com/overhangio/tutor-forum/pull/49/files#diff-7b3ed02bc73dc06b7db906cf97aa91dec2b2eb21f2d92bc5caa761df5bbc168f)).

Users not running Tutor will by default remain on the legacy cs_comments_service app. They need to read this part of the forum app README: https://github.com/openedx/forum?tab=readme-ov-file#forum-v2-toggle Some settings need to be defined even for legacy app users.

If you need more information please get in touch.

Reply:
I added a line pointing to the DEPR issue. If you think more of your descriptive verbage needs to go in please suggest a change and I'll commit it :)

Thank you!